packages/back-end/src/models/ExperimentLaunchChecklistModel.ts

```diff
@@ -0,0 +1,93 @@
+import mongoose from "mongoose";
+import uniqid from "uniqid";
+import { omit } from "lodash";
+import {
+  ChecklistTask,
+  ExperimentLaunchChecklistInterface,
+} from "../../types/experimentLaunchChecklist";
+
+const experimentLaunchChecklistSchema = new mongoose.Schema({
+  id: String,
+  organizationId: String,
+  dateCreated: Date,
+  dateUpdated: Date,
+  updatedByUserId: String,
+  tasks: [
+    {
+      task: String,
+      completionType: { type: String, enum: ["manual", "auto"] },
+      propertyKey: {
+        type: String,
+        enum: ["description", "hypothesis", "project", "tag", "screenshots"],
+      },
+      url: String,
+    },
+  ],
+});
+
+export type ExperimentLaunchChecklistDocument = mongoose.Document &
+  ExperimentLaunchChecklistInterface;
+
+export const ExperimentLaunchChecklistModel = mongoose.model<ExperimentLaunchChecklistInterface>(
+  "ExperimentLaunchChecklist",
+  experimentLaunchChecklistSchema
+);
+
+function toInterface(
+  doc: ExperimentLaunchChecklistDocument
+): ExperimentLaunchChecklistInterface {
+  return omit(doc.toJSON<ExperimentLaunchChecklistDocument>(), ["__v", "_id"]);
+}
+
+export async function createExperimentLaunchChecklist(
+  organizationId: string,
+  createdByUserId: string,
+  tasks: ChecklistTask[]
+): Promise<ExperimentLaunchChecklistInterface> {
+  const doc: ExperimentLaunchChecklistDocument = await ExperimentLaunchChecklistModel.create(
+    {
+      id: uniqid("exp-list-"),
+      organizationId,
+      dateCreated: new Date(),
+      dateUpdated: new Date(),
+      updatedByUserId: createdByUserId,
+      createdByUserId,
+      tasks,
+    }
+  );
+
+  return toInterface(doc);
+}
+
+export async function getExperimentLaunchChecklistByOrgIg(
+  organizationId: string
+): Promise<ExperimentLaunchChecklistInterface | null> {
+  const doc: ExperimentLaunchChecklistDocument | null = await ExperimentLaunchChecklistModel.findOne(
+    {
+      organizationId,
+    }
+  );
+
+  return doc ? toInterface(doc) : null;
+}
```

Comment:
If we want to eventually support multiple checklists, it makes sense to return an array of all matching documents here.  

Right now this model is a weird mix of methods.  You can create as many new checklists in an org as you want and can update by id, but there's no way to get all checklists or get a specific one by id.  There's just a magic method that returns the first random one it finds

Reply:
I've updated this and other areas of the code so an organization can only have a single org-level checklist. 

The idea is, an org-level checklist is a checklist that doesn't contain a `projectId` property.

Likewise, when we add project-level checklist support, I think we should limit each project to a single checklist.

I've updated this to take in an optional `projectId` param, and return a single result based upon that. Likewise, I've updated the POST request on the controller to prevent multiple checklists per org/project.